ports/libsnoretoast/portfile.cmake

```
@@ -0,0 +1,29 @@
+vcpkg_fail_port_install(ON_TARGET "osx" "linux")
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO KDE/snoretoast
+    REF v0.8.0
+    SHA512 233751b6cc3f8099c742e4412a3c9ba8707a2f3c69b57bab93dd83b028aa0c0656cade8de1ece563843ace576fd0d8e5f3a29c254a07ed939d0a69cd2d4f6c2a
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS 
+        -DBUILD_EXAMPLES=OFF

```

```suggestion
        -DBUILD_EXAMPLES=OFF
        -DBUILD_STATIC_RUNTIME=OFF
```
vcpkg will pass the correct `/MT`/`/MD` setting in, so we don't need the library's handling of that: https://invent.kde.org/libraries/snoretoast/-/blob/master/CMakeLists.txt#L23